### PR TITLE
Game rework

### PR DESCRIPTION
**Pull request by EthanDelage**

### CODE REVIEW

**Review by tdameros — APPROVED** [review]

File: front/app/src/js/components/game/Scene/Scene.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ export class Scene {
     const loosersJson = sceneJson['loosers'];
     for (const looserJson of loosersJson) {
       const newLooser = new Player();
-      await newLooser.init(loosersJson);
+      await newLooser.init(looserJson);
```
Comment (hferraud) [inline front/app/src/js/components/game/Scene/Scene.js:48]: Is this correct ? @V-Fries

File: front/app/src/js/components/game/Scene/Player/Paddle.js [hunk 2]
```diff
@@ -91,23 +91,22 @@ export class Paddle {
...
 
-    const xDirectionToLookAt = 999999999999999.;
+    const xDirectionToLookAt = 1.;
     if (this.#paddleIsOnTheRight) {
-      this.#light.position.set(-(this.#paddleSize.x * 0.5), 0., 0.);
+      this.#light.position.set(-(this.#paddleSize.x * 0.5), 0., -0.25);
       this.#light.lookAt(-xDirectionToLookAt, 0., 0.);
     } else {
-      this.#light.position.set(this.#paddleSize.x * 0.5, 0., 0.);
+      this.#light.position.set(this.#paddleSize.x * 0.5, 0., -0.25);
```
Comment (V-Fries) [inline front/app/src/js/components/game/Scene/Player/Paddle.js:109]: What is the -0.25?

File: front/app/src/js/components/game/Scene/Scene.js (hunk 1 above)
Comment (V-Fries) [reply front/app/src/js/components/game/Scene/Scene.js:48]: Looks good

File: front/app/src/js/components/game/Scene/Player/Paddle.js (hunk 2 above)
Comment (EthanDelage) [reply front/app/src/js/components/game/Scene/Player/Paddle.js]: This is because the paddles are not on the ground.

Comment (V-Fries) [reply front/app/src/js/components/game/Scene/Player/Paddle.js]: Shouldn't it be 0.25 since the paddles are at 0.25?

Comment (EthanDelage) [reply front/app/src/js/components/game/Scene/Player/Paddle.js]: yes, I just made the change

**Review by V-Fries — APPROVED** [review]

**Pull request merged**